common/coat_of_arms/coat_of_arms/wc_easter_egg_dynasties.txt

```diff
@@ -0,0 +1,10 @@
+### Humans ###
```

Comment:
The properties file is never yellow. How they work is explained in the Great Big Guide.
https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2/wiki/Model-Guidelines#guidelines
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/126911951-deff94fd-f1d6-4931-84ec-3f5455a8e6a0.png)

</details>